jp2_pc/Source/Lib/EntityDBase/QueueMessage.cpp

```diff
@@ -138,7 +138,7 @@
 		//
 		while (!pdqwmCurrentMessages->empty())
 		{
-			(pdqwmCurrentMessages->front())->Send();
+			pdqwmCurrentMessages->front().front()->Send();
 			pdqwmCurrentMessages->pop();
```

Comment:
`pdqwmCurrentMessages` is a `std::queue` of a `std::deque`, we must use `front()` twice to access the first inner element.